### PR TITLE
chore(releasing): Prepare v0.35.1

### PR DESCRIPTION
**Pull request by jszwedko**
Checklist: https://github.com/vectordotdev/vector/issues/19811

### CODE REVIEW

File: website/cue/reference/releases/0.35.1.cue [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+		"""
+
+	changelog: [
+		{
+			type: "fix"
+			scopes: ["http provider"]
+			description: """
+				The HTTP server-based sources that support the new
+				`keepalive.max_connection_age_secs` now only apply this setting for HTTP/0.9,
+				HTTP/1.0, and HTTP/1.1 connections since the implementation, which sends
```
Comment (buraizu) [inline website/cue/reference/releases/0.35.1.cue:23]:
```suggestion
				HTTP/1.0, and HTTP/1.1 connections, since the implementation, which sends
```

**Review by buraizu — APPROVED** [review]
Looks good overall, approving with one minor punctuation suggestion to clarify a sentence

File: website/cue/reference/releases/0.35.1.cue (hunk 1 above)
Comment (jszwedko) [reply website/cue/reference/releases/0.35.1.cue]: Thanks!

File: changelog.d/19643_only_set_connection_close_http1.fix.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+When terminating idle HTTP connections using the configured `max_connection_age`, only send
+`Connection: Close` for HTTP/0.9, HTTP/1.0, and HTTP/1.1 requests. This header is not supported on
+HTTP/2 and HTTP/3 requests. This may be supported on these HTTP versions in the future.
```
Comment (neuronull) [inline changelog.d/19643_only_set_connection_close_http1.fix.md:3]: 👀  did the generate script not pull this one in? 😟

Comment (jszwedko) [reply changelog.d/19643_only_set_connection_close_http1.fix.md:3]: Ah, v0.35.0 was before the tooling for generating changelogs existed so I manually created this one.

**Review by neuronull — APPROVED** [review]

**Comment by jszwedko** [comment]
Merged manually to preserve commit history.

**Pull request closed without merging**